.wwhrd.yml

```diff
@@ -0,0 +1,14 @@
+---
+blacklist:
+  - GPL-2.0
```

Comment:
why are we blacklisting GPL-2.0?

Reply:
https://www.apache.org/licenses/GPL-compatibility.html
Please note,
The lists are expected to grow eventually :)
This PR is an attempt to get the base framework